packages/meditrak-server/src/routes/accessRequests/assertAccessRequestPermissions.js

```diff
@@ -12,3 +12,22 @@ export const assertAccessRequestPermissions = async (accessPolicy, models, acces
   }
   return true;
 };
+
+export const createAccessRequestDBFilter = async (accessPolicy, models, criteria) => {
+  if (hasBESAdminAccess(accessPolicy)) {
+    return criteria;
+  }
+  // If we don't have BES Admin access, add a filter to the SQL query
+  const dbConditions = criteria;
+  const accessibleCountryCodes = accessPolicy.getEntitiesAllowed(
+    TUPAIA_ADMIN_PANEL_PERMISSION_GROUP,
+  );
+  accessibleCountryCodes.push('DL'); // If we have admin panel anywhere, we can also view Demo Land
```

Comment:
This looks like it might return demo land info for people who have no admin panel access at all?

Reply:
You wouldn't get to here without any admin panel access because of how we gate route handlers with `assertUserHasAccess`

Reply:
Right, that makes sense. However, I think we should have that logic here as well - coupling logic in different files could be dangerous, especially when this is a nicely named util that someone might pick up to use somewhere else.

Reply:
Added a length check to ensure we have __some__ permissions before adding demo land